astropy/io/ascii/latex.py

```diff
@@ -87,7 +87,7 @@ def find_latex_line(lines: list[str], latex: str) -> int | None:
         Line number. Returns None, if no match was found
 
     """
-    re_string = re.compile(latex.replace("\\", "\\\\"))
+    re_string = re.compile(r"\s*" + latex.replace("\\", "\\\\"))
```

Comment:
As you're touching this line anyway, maybe also raw strings for the replacement (`.replace(r"\", r"\\")`)?

Reply:
Sure. Not much of a difference in terms of number of characters or readability, but why not?

Reply:
It turned out that this replacement was the problem with the indentation, so I removed it again and force-pushed to your branch. Not worth thinking about too much, but my emacs setup for colouring python code could not `r"\\"`, and I suspect it is the same for pre-commit...